gym/wrappers/__init__.py

```diff
@@ -1,3 +1,4 @@
+"""Module of wrapper functions."""
```

Comment:
Pedantic remark, but I would argue that it's mostly classes

Reply:
Very right, changed